docs/core_library/jspsych-core.md

```diff
@@ -363,7 +363,7 @@ use_webaudio | boolean | If false, then jsPsych will not attempt to use the WebA
 default_iti | numeric | The default inter-trial interval in ms. The default value if none is specified is 0ms.
 experiment_width | numeric | The desired width of the jsPsych container in pixels. If left undefined, the width will be 100% of the display element. Usually this is the `<body>` element, and the width will be 100% of the screen size.
 minimum_valid_rt | numeric | The minimum valid response time for key presses during the experiment. Any key press response time that is less than this value will be treated as invalid and ignored. Note that this parameter only applies to _keyboard responses_, and not to other response types such as buttons and sliders. The default value is 0.
-override_safe_mode | boolean | If jsPsych detects that the HTML file has been opened directly in the browser, rather than running via a server, then it will disable Web Audio and video preloading in order to prevent cross-origin request (CORS) errors. Default is `true`. If `false`, jsPsych will ignore the file's protocol and use the normal settings for Web Audio (`use_webaudio`) and video preloading (automatic preloading via video plugins and/or manual preloading via `preload_videos`).
+override_safe_mode | boolean | If jsPsych detects that the HTML file has been opened directly in the browser, rather than running via a server, then it will disable Web Audio and video preloading in order to prevent cross-origin request (CORS) errors. Default is `false`. If `true`, jsPsych will ignore the file's protocol and use the normal settings for Web Audio (`use_webaudio`) and video preloading (automatic preloading via video plugins and/or manual preloading via `preload_videos`).
```

Comment:
I've been thinking about how to make it a little clearer what the purpose of the parameter is. What do you think about this edit?

override_safe_mode | boolean | Opening a jsPsych experiment in a web browser on a computer (e.g., by double clicking on an HTML file on the hard drive) will load the page using the `file://` protocol. Some features of jsPsych don't work with this protocol. By default, when jsPsych runs on a page loaded via the `file://` protocol it runs in *safe mode*, which disables features that don't work in this context. Specifically, the use of Web Audio is disabled (Audio will be played using HTML5 audio instead) and video preloading is disabled. You can set this parameter to `true` to force these features to operate under the `file://` protocol. In order to get this to work, you will need to disable web security features in your browser. This is safe to do if you know what you are doing. Note that this parameter has no effect when you are running the experiment on a web server, because the page will be loaded via the `http://` or `https://` protocol.

Reply:
Yep that's clearer. We'll explain this in the other more narrative docs as well, but probably helps to explain it clearly in both places.

Reply:
I pushed this change but tweaked the wording a bit. I think my version only shows up in the file diff section. Please let me know if you want any changes. Thanks!!

Reply:
Looks great!